internal/tests/testrepository/header_testrepository.go

```diff
@@ -193,6 +194,49 @@ func (r *HeaderTestRepository) GetChainBetweenTwoHashes(low string, high string)
 	return headers, nil
 }
 
+func (r *HeaderTestRepository) GetMerkleRoots(batchSize int, lastEvaluatedKey int) (*domains.MerkleRootsESKPagedResponse, error) {
```

Comment:
Please remember about comment, linter settings forces that 🤷

Reply:
Added comment